pymongo/message.py

```diff
@@ -443,29 +457,23 @@ def get_message(self, dummy0, sock_info, use_cmd=False):
 
 
 class _RawBatchQuery(_Query):
-    def use_command(self, socket_info, exhaust):
-        # Compatibility checks.
-        super(_RawBatchQuery, self).use_command(socket_info, exhaust)
-        # Use OP_MSG when available.
-        if socket_info.op_msg_enabled and not exhaust:
-            return True
-        return False
-
-    def get_message(self, set_slave_ok, sock_info, use_cmd=False):
-        return super(_RawBatchQuery, self).get_message(
-            set_slave_ok, sock_info, use_cmd)
+    pass
+    # def use_command(self, socket_info, exhaust):
+    #     # Compatibility checks.
+    #     super(_RawBatchQuery, self).use_command(socket_info, exhaust)
+    #     # Use OP_MSG when available.
+    #     if socket_info.op_msg_enabled:
+    #         return True
+    #     return False
```

Comment:
So these classes do nothing now? Can we remove them?

Reply:
Done.